### PR TITLE
Update website content

### PR DESCRIPTION
**Pull request by momegas**
* Add content for RnD page
* Remove open banking content
* Add new content according to services

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   siteMetadata: {
-    title: 'Data analytics & Open Banking',
+    title: 'Homepage | Squaredev',
```
Comment (ksiabani) [inline gatsby-config.js:3]: I think this is shown to every page, can you check? We don't want to show `Homepage` to every page

File: src/components/footer/footer.js [hunk 2]
```diff
@@ -86,32 +86,26 @@ const Footer = () => (
...
+              </a>
+            </div>
+          </div>
         </nav>
       </div>
       <div className="row">
         <div className="footer__text col-12">
           Copyright © {new Date().getFullYear()} Squaredev, All rights reserved
-          | Cookies | Policy
+          {/* | Cookies | Policy */}
```
Comment (ksiabani) [inline src/components/footer/footer.js:108]: Thanks!

Comment (momegas) [reply src/components/footer/footer.js:108]: Welcome

File: src/pages/company/1-hero.md [hunk 3]
```diff
@@ -13,9 +13,9 @@
 ##
 template: default
 component: hero
-heading:  Our mission is to build the trust, between lenders and borrowers, from the ground up.
-description: We are a diverse team of youth and experience, that uses data science with Open Banking (PSD2) to build predictive analytics solutions for all.
-image: bright-yellow-brick.jpg
+heading: We use beyond state of the art technology to improve peoples' lives
+description: We use our technologies for widespread public benefit and scientific discovery
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/company/1-hero.md:17]: Changed

File: src/pages/home/1-banner.md [hunk 4]
```diff
@@ -1,23 +1,21 @@
...
 ##
 template: default
 component: banner
-headline: We are a data analytics startup
-heading: We believe technology should improve lives
-btnTxt: More on solutions
-btnUrl: cards4
+headline: Innovate. Thats what we do
+heading: We use beyond state of the art technology to improve peoples' lives
+btnTxt: More on RnD
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/home/1-banner.md:17]: changed

File: src/pages/home/2-ai-applied.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+# imagePosition: The position of the image (can be "left" or "right", default is "left")
+# backgroundSize: How image is rendered as background (can be "contain" or "cover", default is "cover")
+##
+template: default
+component: imageText
+headline: _ Artificial intelligence
+heading: AI and data science applied in some of the hardest problems
+btnTxt: Find out more
+btnUrl: research
+image: flow-mac.jpg
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/home/2-ai-applied.md:23]: Chose a chip image :P

File: src/pages/home/2-ai-applied.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+imagePosition: left
+backgroundSize: cover
+---
+
+We apply our AI research in numerous fields. Cancer detection and classification, pandemics and linked open data are some of the fields we are working on to apply our findings.
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/home/2-ai-applied.md:28]: Changed to: Diagnosis

File: src/pages/home/1-banner.md [hunk 7]
```diff
@@ -1,23 +1,21 @@
...
+# btnUrl: The link to be followed when button is clicked
 ##
 template: default
 component: banner
-headline: We are a data analytics startup
-heading: We believe technology should improve lives
-btnTxt: More on solutions
-btnUrl: cards4
+headline: Innovate. Thats what we do
+heading: We use beyond state of the art technology to improve peoples' lives
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/home/1-banner.md:16]: Changed to: We use technology to improve peoples' lives

File: src/pages/home/3-solutions.md [hunk 8]
```diff
@@ -23,35 +23,50 @@ icon: lab-tube-bold-green.png
 ##
 ---
 
-# Credit scoring
-## Achieve higher acceptance rates and fewer bad loans, harnessing Open Banking data. 
-### user-credit-card-bold-green.png
+# Artificial intelligence
+
+## Human brains are great, but as data gets bigger, they too, need help. AI is the answer to large data problems
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/home/3-solutions.md:28]: Cut some words ;)

File: src/pages/research/2-grid.md [hunk 9]
```diff
@@ -19,30 +19,32 @@ heading: Several different domains with a common denominator. Our technology. Kn
...
 
-# Data Analytics & Information Visualization
-## single-man-bold-red.png
+# Explainable AI
 
-**Privacy-preserving Platform (PPP)** for GDPR compliance ensuring that your data is remembered and respected. 
+## user-360-bold-green.png
 
----
+AI decisions have remained a black-box for a long time. We try to solve this by building expandability in systems. All our AI systems use expandability at their core. One of our research projects [INCISIVE](/research#imagetext4) is focused on classifying cancer images and explaining the patterns found to the user for better understanding. Reports produced in [Wysely](/research#imagetext3) provide a holistic assessment explained to risk officers.
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/research/2-grid.md:26]: Changed

File: src/pages/research/2-grid.md [hunk 10]
```diff
@@ -19,30 +19,32 @@ heading: Several different domains with a common denominator. Our technology. Kn
...
 ---
 
-# Smart Technologies
-## maps-search-bold-green.png
+# Natural Language Classification
+
+## robot-head-bold-green.png
 
-**IoT Device Behaviour Analyzer (DBA)** for automatic detection of suspicious (unauthorized) devices connected to your IoT network.
+We use our **Natural Language Classification Engine** in [Wysely](/research#imagetext3) for transaction classification. This results in contextually understanding data and providing accurate, actionable insights that are explainable to the end-user.
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/research/2-grid.md:42]: Thenged to lower case

File: src/pages/research/2-grid.md [hunk 11]
```diff
@@ -19,30 +19,32 @@ heading: Several different domains with a common denominator. Our technology. Kn
...
 ---
 
-# Safety & (Cyber)Security
-## shopping-cart-bold-green.png
-**Data Lineage Platform (DLP)** for improving your organization’s transparency with regulators regarding personal data flows.
+# Graphs at Scale & Linked Data
+
+## databases-bold-green.png
+
+At [STAMINA](/research#imagetext5), one of out research projects, we scout the web and and semantically link open data with our **Natural Language Classification Engine** creating a large graph. Then **Graph Data Science Techniques** are used on the data to provide insights and alerts.
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/research/2-grid.md:50]: Same as above

File: src/pages/research/3-wysely.md [hunk 12]
```diff
@@ -1,30 +1,28 @@
...
 ##
 template: default
 component: imageText
 headline: _ Open banking
-heading: Transform your lending and scoring with Open Banking
-btnTxt: Let's talk
-btnUrl: company/#contact2
-image: flow-mac.jpg
+heading: Wysely - wiser decisions for lenders and borrowers
+btnTxt: Visit Wysely
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/research/3-wysely.md:21]: Done

File: src/pages/research/4-incisive.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+# btnUrl: The link to be followed when button is clicked
+# image: The image to use
+# imagePosition: The position of the image (can be "left" or "right", default is "left")
+# backgroundSize: How image is rendered as background (can be "contain" or "cover", default is "cover")
+##
+template: default
+component: imageText
+headline: _ Cancer
+heading: INCISIVE - An AI-powered cancer image repository for diagnosis, prediction and follow-up
+btnTxt: Visit INCISIVE
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/research/4-incisive.md:21]: Done

File: src/pages/research/4-incisive.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+# heading: The title of the section
+# btnTxt: The text of the button, if left blank, button will be hidden
+# btnUrl: The link to be followed when button is clicked
+# image: The image to use
+# imagePosition: The position of the image (can be "left" or "right", default is "left")
+# backgroundSize: How image is rendered as background (can be "contain" or "cover", default is "cover")
+##
+template: default
+component: imageText
+headline: _ Cancer
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/research/4-incisive.md:19]: Replaces

File: src/components/header/header.js [hunk 15]
```diff
@@ -66,71 +66,68 @@ const Header = ({ isScrolled, isMobileMenuVisible, toggleMobileMenu }) => (
...
           <ul className="nav-menu sf-arrows">
-            {/*<li>*/}
-            {/*  <Link to="/research">Research</Link>*/}
-            {/*</li>*/}
+            <li>
+             <Link to="/research">Research</Link>
+            </li>
             <li className="menu-has-children">
-              <h5 className="menu-has-children__title">Solutions</h5>
+              <h5 className="menu-has-children__title">Technology</h5>
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/components/header/header.js:73]: Commented in code for later use

File: src/pages/home/3-solutions.md [hunk 16]
```diff
@@ -11,7 +11,7 @@
 ##
 template: default
 component: cards
-heading: Our data solutions
+heading: Research & dev
```
[the remark this answers is not in the document]
Comment (momegas) [reply src/pages/home/3-solutions.md:14]: Changed

File: src/pages/home/3-solutions.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (ksiabani) [reply src/pages/home/3-solutions.md]: Very nice!

**Pull request merged**